frontend/src/slices/auth/actions.ts

```diff
@@ -42,6 +42,16 @@ const signIn = createAsyncThunk<
   return user;
 });
 
+const signOut = createAsyncThunk<null, undefined, AsyncThunkConfig>(
+  `${sliceName}/sign-out`,
+  async (_, { extra }) => {
+    const { storage } = extra;
+    await storage.drop(StorageKey.TOKEN);
+
+    return null;
```

Comment:
we don't need to return anything here
```suggestion
```

Reply:
Initially, I wanted to do so, but without any return statement I get an error. Couldn't find a right solution for it. 
So I tried following the same approach that followed every other action - return something even if it's null: 
<img width="323" alt="Screenshot 2023-09-11 at 20 55 35" src="https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/62558753/287b891d-f2d3-418e-ba48-2493259404fd">
<img width="329" alt="Screenshot 2023-09-11 at 20 55 47" src="https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/62558753/c978a990-1ade-41a9-b873-b8594a4efaea">
<img width="776" alt="Screenshot 2023-09-11 at 20 55 57" src="https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/62558753/878fe2aa-feda-4cc5-80f3-0b1f707e1cfa">

Reply:
pass unknown istead of null as a first parameter of generic